webrender/src/resource_cache.rs

```diff
@@ -116,6 +116,40 @@ struct ImageResource {
     dirty_rect: Option<DeviceUintRect>
 }
 
+struct ImageTemplates {
```

Comment:
as per IRC discussion, having a separate type allows us to implement `ImageStore`, which is needed to abstract away from `BlobImageRenderer`